recipes/centos_spring_boot/configure_maven_and_deps.sh

```diff
@@ -21,6 +22,21 @@ function install {
   fi
 }
 
+# insert settings for mirrors/repository managers into settings.xml if supplied
+function configure_mirrors {
+  if [ -n "$MAVEN_MIRROR_URL" ]; then
+    xml="    <mirror>\
+      <id>mirror.default</id>\
+      <url>$MAVEN_MIRROR_URL</url>\
+      <mirrorOf>external:*</mirrorOf>\
+    </mirror>"
+    sed -i "s|<!-- ### configured mirrors ### -->|$xml|" $HOME/.m2/settings.xml
+  fi
+}
+
+# check for the MAVEN_MIRROR_URL env variable, if its available then set maven mirrors in $HOME/.m2/settings.xml
+configure_mirrors
+# TODO - do we really need to do this ? 
```

Comment:
why do we have a TODO here ?

Reply:
@benoitf - i really don't know why the dependencies are installed, when application needs it can be pulled automatically. WDYT ? if you are fine we can remove install steps from all similar stack docker images

Reply:
I would say that it's not because you don't know that we should add a TODO comment.
And I'm not fine about removing these steps as they were added to make the experience better.